src/parsers/manifest/dash/parse_adaptation_sets.ts

```diff
@@ -97,8 +98,13 @@ function isVisuallyImpaired(
     return false;
   }
 
-  return (accessibility.schemeIdUri === "urn:tva:metadata:cs:AudioPurposeCS:2007" &&
+  const isVisuallyImpairedAudioDvbDash = (accessibility.schemeIdUri === "urn:tva:metadata:cs:AudioPurposeCS:2007" &&
```

Comment:
The indentation of the following line now looks off.

It has:
  - either too many spaces if you're just 2-space indenting or
  - not enough if you're aligning with the related block of code in the previous line

We usually go with the latter strategy, though not always and you could do one or the other, as looks best to you.